configuration/configuration.go

```diff
@@ -172,6 +172,15 @@ type Configuration struct {
 			TrustKey string `yaml:"signingkeyfile,omitempty"`
 		} `yaml:"schema1,omitempty"`
 	} `yaml:"compatibility,omitempty"`
+
+	Validation struct {
```

Comment:
I believe the linter doesn't care about documentation for nested structs, only structs defined at the top level.